diff --git a/node_modules/react-native-paper/lib/typescript/components/Typography/Text.d.ts b/node_modules/react-native-paper/lib/typescript/components/Typography/Text.d.ts
index 035d9b5..cb32017 100644
--- a/node_modules/react-native-paper/lib/typescript/components/Typography/Text.d.ts
+++ b/node_modules/react-native-paper/lib/typescript/components/Typography/Text.d.ts
@@ -1,5 +1,4 @@
 import * as React from 'react';
-import { ReactNode } from 'react';
 import { StyleProp, Text as NativeText, TextStyle } from 'react-native';
 import type { VariantProp } from './types';
 import type { ThemeProp } from '../../types';
@@ -28,9 +27,7 @@ export type Props<T> = React.ComponentProps<typeof NativeText> & {
 export type TextRef = React.ForwardedRef<{
     setNativeProps(args: Object): void;
 }>;
-type TextComponent<T> = (props: Props<T> & {
-    ref?: React.RefObject<TextRef>;
-}) => ReactNode;
+type TextComponent<T> = React.FC<Props<T> & { ref?: React.RefObject<TextRef> }>;
 declare const Component: TextComponent<never>;
 export declare const customText: <T>() => TextComponent<T>;
 export default Component;
